parcels/kernel.py

```diff
@@ -116,6 +117,7 @@ def __init__(self, fieldset, ptype, pyfunc=None, funcname=None,
             try:
                 user_ctx = stack[-1][0].f_globals
                 user_ctx['math'] = globals()['math']
+                user_ctx['ParcelsRandom'] = globals()['ParcelsRandom']
                 user_ctx['random'] = globals()['random']
```

Comment:
why is this then still required ? What kind of test breaks when removing this line ? to my understanding, those stack-parses are only required because of JIT, JIT should always use ParcelsRandom, so from the current viewpoint I don't see why this is still here. Maybe I just check out the PR and see it myself, for curiosity

Reply:
I could indeed remove `user_ctx['random'] = globals()['random']`, but then users won't be able to concat python 'random' functions in scipy mode (note that this all applies to scipy kernel concatenation). For backward compatibility, I decided to keep it